frontend/src/scenes/insights/PersonModal.js

```diff
@@ -34,7 +32,7 @@ export function PersonModal({ visible, view }) {
                     style={{ marginBottom: 16, display: 'flex', alignItems: 'center', justifyContent: 'space-between' }}
                 >
                     Found {people.count === 99 ? '99+' : people.count} {people.count === 1 ? 'user' : 'users'}
-                    {user?.is_multi_tenancy && featureFlags['filter_by_session_props'] ? (
+                    {featureFlags['filter_by_session_props_link'] ? (
```

Comment:
Context on why the switch in the feature flag?

Reply:
This way we can turn this feature on but not the new filters -> I think this is already useful as-is.